File: ChatBot/app/src/main/java/kumar/prince/chatbot/InitApp.java
package kumar.prince.chatbot;

import android.app.Application;

import com.google.firebase.database.FirebaseDatabase;

/**
 * Created by prince on 21/3/18.
 */

public class InitApp extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        FirebaseDatabase.getInstance().setPersistenceEnabled(true);
    }
}

File: Androidkotlin/app/src/main/java/kumar/prince/kotlinapp/InitApp.kt
package kumar.prince.kotlinapp

import android.app.Application
import android.content.Context
import android.util.Log

/**
 * Created by prince on 6/3/18.
 */
class InitApp : Application(){
    var cont: Context?=null

    private val tag: String?="InitApp"

    override fun onCreate() {
        super.onCreate()
        cont=applicationContext
        Log.v(tag,"On Create App")
    }
}